src/cmd/cli/command/commands.go

```diff
@@ -360,6 +360,16 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		// Start auth server for docker login flow if port is specified
+		port, _ := cmd.Flags().GetInt("auth-server")
+		if port != 0 {
+			go func() {
+				if err := cli.InteractiveLoginWithDocker(cmd.Context(), getCluster(), port); err != nil {
```

Comment:
only the MCP server needs this to be long running. Regular CLI can use the existing flow (Start/stop server when needed) but honoring the `--auth-server` port that was given.